src/main/java/monday/monday/exception/MondayExceptions.java

```diff
@@ -0,0 +1,27 @@
+package monday.monday.exception;
+
+/**
+ * MondayExceptions is a custom exception class for handling
+ * exceptions related to the Monday application.
+ */
+public class MondayExceptions extends Exception {
```

Comment:
Since the class refers to a single instance of an exception, perhaps it should be named `MondayException` (singular) instead?